libraries/Wire/Wire.cpp

```diff
@@ -20,11 +20,8 @@ TwoWire::TwoWire(cyhal_gpio_t sda, cyhal_gpio_t scl, uint8_t instance) : sda_pin
 }
 
 void TwoWire::_begin() {
-    rxBufferIndex = 0;
-    rxBufferLength = 0;
-
-    txBufferIndex = 0;
-    txBufferLength = 0;
+    rxBuffer.clear();
```

Comment:
I would expect that the buffers are clean before begin(), either due to first time initiliazation or due to user calling .end() or .flush() before calling begin again.

Reply:
Done